### DISCUSSION TITLE
Participation Status in the Competition Tracks of SBFT and ICST

### DISCUSSION THREAD

**Discussion by ali-ihsan-gullu** [christianbirchler-org/sdc-testing-competition#33] [Q&A] [answered]
Can the same tool submitted for ICST also be submitted for SBFT? Is it possible to compete in both tracks with the same tool? If not, would it be acceptable to participate in SBFT with a slightly modified version, or is it necessary to develop a completely different tool based on a new method?

**Comment by spanichella** [comment, marked as answer]
A modified version of the tool is sufficient, but the differences have to be clarified in the submission time (and the eventual too reports). Also the naming of the tool should be changed, to reflect it.
